modules/configuring-firewall.adoc

```diff
@@ -46,14 +46,13 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`
 
-|`rhcos.mirror.openshift.com`
+|`rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com`
```

Comment:
4.11+ will only use the new redirector rhcos.mirror.openshift.com.  You can remove this entry

Reply:
thanks, updated!